webapp/src/extensionsBrowser.tsx

```diff
@@ -448,18 +448,26 @@ export const ExtensionsBrowser = (props: ExtensionsProps) => {
                         }
                         {displayMode == ExtensionView.Tabbed &&
                             <div className="tab-header">
-                                <Button
-                                    title={lf("Recommended")}
-                                    label={lf("Recommended")}
-                                    onClick={() => { setCurrentTab(TabState.Recommended) }}
-                                    className={currentTab == TabState.Recommended ? "selected" : ""}
-                                />
-                                <Button
-                                    title={lf("In Development")}
-                                    label={lf("In Development")}
-                                    onClick={() => { setCurrentTab(TabState.InDevelopment) }}
-                                    className={currentTab == TabState.InDevelopment ? "selected" : ""}
-                                />
+                                {extensionsInDevelopment.length > 0 &&
+                                    <>
+                                        <Button
+                                            key={"Recommended"}
```

Comment:
only needed when using maps and creating tree elements in a loop

Reply:
Ah, these were removed in and older PR and shouldn't be here. Diff isn't as good when whole blocks of code are being moved 😅.